migrations/contracts-1.1/src/storage-wrappers/minter-contract.ts

```diff
@@ -0,0 +1,105 @@
+import { TezosToolkit } from "@taquito/taquito";
+import { substituteVariables } from "./wrapper-utils";
+
+type MinterStorage = {
+  collateralizationPercentage: number
+  developerFundContractAddress: string
+  governorContractAddress: string
+  interestIndex: number
+  lastInterestIndexUpdateTime: any
+  liquidationFeePercent: number
+  ovenProxyContractAddress: string
+  devFundSplit: number
+  stabilityFee: number
+  stabilityFundContractAddress: string
+  tokenContractAddress: string,
+  liquidityPoolContractAddress: string,
+  privateOwnerLiquidationThreshold: number
+}
+
+export async function generateMinterStorage(
+  oldMinterContractAddress: string,
+  liquidityPoolContractAddress: string,
+  governorContractAddress: string,
+  privateOwnerLiquidationThreshold: number,
+  config: any
+) {
+  // Grab old values from storage
+  const tezos: TezosToolkit = await config.getTezos()
+  const oldMinterContract = await tezos.contract.at(oldMinterContractAddress)
+  const oldMinterStorage: any = await oldMinterContract.storage()
+
+  const collateralizationPercentage = oldMinterStorage.collateralizationPercentage
+  const developerFundContractAddress = oldMinterStorage.developerFundContractAddress
+  const interestIndex = oldMinterStorage.interestIndex
+  const lastInterestIndexUpdateTime = new Date(oldMinterStorage.lastInterestIndexUpdateTime).getTime() / 1000
+  const liquidationFeePercent = oldMinterStorage.liquidationFeePercent
+  const ovenProxyContractAddress = oldMinterStorage.ovenProxyContractAddress
+  const devFundSplit = oldMinterStorage.stabilityDevFundSplit
+  const stabilityFee = oldMinterStorage.stabilityFee
+  const stabilityFundContractAddress = oldMinterStorage.stabilityFundContractAddress
+  const tokenContractAddress = oldMinterStorage.tokenContractAddress
+
+  // Formulate a new storage
+  let contractStorage = `
+  (Pair 
+    (Pair 
+      (Pair 
+        (nat %collateralizationPercentage) 
+        (Pair 
+          (nat %devFundSplit) 
+          (address %devFundAddress)
+        )
+      ) 
+      (Pair 
+        (address %governorAddress)
+        (Pair 
+          (nat %interestIndex) 
+          (timestamp %lastInterestIndexUpdateTime)
+        )
+      )
+    ) 
+    (Pair 
+      (Pair
+        (nat %liquidationFeePercent) 
+        (Pair 
+          (address $liquidityPoolAddress) 
```

Comment:
I think this should be `%liquidityPoolAddress` (though it's correctly replacing currently below, so don't forget to update that too!)